Tests/AK/TestAKMath.cpp

```diff
@@ -30,5 +30,5 @@ TEST_CASE(pow)
     EXPECT_EQ(AK::pow(1., 9223372036854774784.0), 1.);
     // Should not take the integer fast path anymore, since the exponent is beyond 64-bit integers.
     EXPECT_EQ(AK::pow(1., 9223372036854775808.0), 1.);
-    EXPECT_EQ(AK::pow(1., 9223372036854775810.0), 1.);
+    EXPECT_EQ(AK::pow(1., 9223372036854777856.0), 1.);
```

Comment:
Why this change in this commit?

Reply:
because rebasing old commits is hard, ugh